src/vector/sinks/metrics.rs

```diff
@@ -46,153 +45,116 @@ impl MetricSink {
         let options = Options::default().credentials_options(credentials_path);
 
         info!("gRPC Options: {:?}", options);
-        let client = Client::create(options).await?;
+        let client = Client::new().await?;
```

Comment:
Is `options` no longer used? How are we making a call using the credentials?

Reply:
The options are now used there: https://github.com/EventStore/nexus/pull/24/files#diff-4aba9d8363f85084cf6274fcf314076ebd1766715752d22e943df7eb85af99a4R142